data/json/items/comestibles/brewing.json

```diff
@@ -497,7 +497,7 @@
     "id": "young_yeast",
     "looks_like": "yeast",
     "name": { "str": "growing wild yeast" },
-    "description": "Mixture of water, sugar and fruits with some naturally occurring yeast, that will eventually produce a more sizeable yeast culture.",
+    "description": "Mixture of water, sugar and fruits with some naturally occurring yeast. If placed in a fermenting vat and given time, it will eventually produce a more sizeable yeast culture.",
```

Comment:
```suggestion
    "description": "Mixture of water, sugar and fruits with some naturally occurring yeast.  If placed in a fermenting vat and given time, it will eventually produce a more sizeable yeast culture.",
```